includes/CVUniverse.cxx

```diff
@@ -1165,7 +1177,7 @@ TVector3 CVUniverse::AdlerAngle(int RefSystemDef, double dmumom /*GeV*/,
   return AdlerSyst;
 }
 
-double CVUniverse::CalcQ2(const double Enu, const double Emu,
+/*double CVUniverse::CalcQ2(const double Enu, const double Emu,
```

Comment:
this is a relevant aaron test. it should be implemented in the new version of this branch.